<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwitchItemViewWidget</class>
 <widget class="QWidget" name="SwitchItemViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>86</width>
    <height>59</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="viewLayout">
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Group</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="itemWidget" native="true">
     <layout class="QVBoxLayout" name="itemLayout"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
